src/OpenFeatureSDK/Model/HookContext.cs

```diff
@@ -65,5 +65,17 @@ public class HookContext<T>
             this.ProviderMetadata = providerMetadata ?? throw new ArgumentNullException(nameof(providerMetadata));
             this.EvaluationContext = evaluationContext ?? throw new ArgumentNullException(nameof(evaluationContext));
         }
+
+        internal HookContext<T> WithNewEvalContext(EvaluationContext context)

```

Comment:
Does it make sense to shorten this? Every where else we have referred to it in full
```suggestion
        internal HookContext<T> WithNewEvaluationContext(EvaluationContext context)
```

Reply:
I think it makes sense to use the full name. Will commit your suggestion.